src/less/progress-spinner/base/progress-spinner.less

```diff
@@ -20,6 +20,17 @@
 }
 
 .spinner--large {
+    background-image: url(@spinner-large-image-url);
     height: 60px;
     width: 60px;
 }
+
+@media (prefers-color-scheme: dark) {
```

Comment:
I wasn't sure how we wanted to manage dark mode with the progress spinner, let me know what I should do here.

As is this works, but I'm not sure if that's how we're managing things at present.

Reply:
Look at any of the ds6 properties files. That's where we manage dark mode.

Example: https://github.com/eBay/skin/blob/master/src/less/properties/ds6/switch-properties.less